test/unit/plugins/provisioners/docker/config_test.rb

```diff
@@ -137,4 +138,25 @@
       })
     end
   end
+
+  describe "#post_install_provision" do
+    it "raises an error if 'docker' provisioner was provided" do
+      expect {subject.post_install_provision("myprov", :type=>"docker", :inline=>"echo 'hello'")}
+        .to raise_error()
+    end
+
+    it "setups a basic provisioner" do
+      prov = double()
+      mock_provisioner = "mock"
+      mock_provisioners = [mock_provisioner]
+
+      allow(VagrantPlugins::Kernel_V2::VMConfig).to receive(:new).
+        and_return(prov)
+      allow(prov).to receive(:provision).and_return(mock_provisioners)
+      allow(prov).to receive(:provisioners).and_return(mock_provisioners)
+
+      subject.post_install_provision("myprov", :inline=>"echo 'hello'")
+      expect(subject.post_install_provisioner).to eq(mock_provisioner)
+    end
```

Comment:
Should include a test with no `post_install_provision` defined to validate correct behavior.